sdk/python/feast/repo_operations.py

```diff
@@ -219,8 +219,8 @@ def plan(repo_config: RepoConfig, repo_path: Path, skip_source_validation: bool)
     click.echo(infra_diff.to_string())
 
 
-def _prepare_registry_and_repo(repo_config, repo_path):
-    store = FeatureStore(config=repo_config)
+def _prepare_registry_and_repo(repo_config, repo_path, is_feast_apply=False):
```

Comment:
I added this logic, because `_prepare_registry_and_repo` is called from `plan` and `apply_total`. I think we want to differentiate where this function was called from.